src/uct/tcp/tcp_ep.c

```diff
@@ -697,11 +697,8 @@ unsigned uct_tcp_ep_progress_rx(uct_tcp_ep_t *ep)
     if (!uct_tcp_ep_ctx_buf_need_progress(&ep->rx)) {
         ucs_assert(ep->rx.buf == NULL);
 
-        ep->rx.buf = ucs_mpool_get_inline(&iface->rx_mpool);
-        if (ucs_unlikely(ep->rx.buf == NULL)) {
-            ucs_warn("tcp_ep %p: unable to get a buffer from RX memory pool", ep);
-            return 0;
-        }
+        UCT_TL_IFACE_GET_RX_DESC(&iface->super, &iface->rx_mpool,
```

Comment:
this macro makes `VALGRIND_MAKE_MEM_DEFINED(_desc, sizeof(*(_desc)));`
How would it work with `ep->(tx|rx).buf`, wich are `void*`?

Reply:
good catch, I don't know how does it work, the same is in UGNI/UDT

it will be fixed shortly